controlplane/pkg/apis/remote/connection/constants.go

```diff
@@ -0,0 +1,8 @@
+package connection
+
+// Remote connection constants
+const (
+	VXLANSrcIP = "srcIp"
```

Comment:
At some point we should discuss whether to use camel case or _ for parameters names for Mechanisms.  This is fine for now, but before the arteries harden we should get to consistency :)

Reply:
Example: https://github.com/ligato/networkservicemesh/blob/master/controlplane/pkg/apis/connectioncontext/constants.go#L4

Reply:
`src_ip` works for me, thanks for pointing this.